bolt/outpost/settlement/v1/settlement.proto

```diff
@@ -265,25 +285,73 @@ message SwapExactInResponse {
 message WithdrawQuotesRequest {
   // base_asset: The base asset against which the accumulated quotes that the user wants to withdraw. e.g. USDT pool
   string base_asset = 1;
-}
-
-// WithdrawQuotesResponse: Response to the WithdrawQuotesRequest. Contains information about the withdrawn quotes.
+  // deposit_address: The address where withdrawn quotes should be deposit. If not provided, the default sender address
+  // would be used. Note it can be overwritten for specific quotes.
+  optional string deposit_address = 2;
```

Comment:
Maybe we should rename this to receiver to match the existing field in SwapExactIn
https://github.com/phi-labs-ltd/bolt-proto/blob/d8043eeec51ac5842447f8b28967d53c1fe81260/bolt/outpost/settlement/v1/settlement.proto#L269

Reply:
Okay, makes sense - will fix. I named it to match the CEX name, but internal protocol consistence is more important.